axon-framework/events/event-processors/README.md

```diff
@@ -331,12 +331,12 @@ public class DeadLetterQueueExampleConfig {
 public class DeadLetterQueueExampleConfig {
     
     @Autowired 
```

Comment:
This should be `@Bean` instead of `@Autowired`.
AF will look for any `ConfigurerModule` beans and tie them into the configuration process.

The `@Autowired` approach (with `void` return type) _used_ to work, but caused cyclic dependencies after some recent update of Spring.

```suggestion
    @Bean 
```